babel.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+	plugins: [
+		'@babel/plugin-syntax-dynamic-import',
+		['@babel/plugin-proposal-class-properties', { loose: true }],
```

Comment:
```suggestion
```
You're not using the class so you don't need this one :)

Same for the dynamic loading, you're not using async javascript loading :)